sanic/request.py

```diff
@@ -197,6 +197,21 @@ async def respond(
         headers: Optional[Union[Header, Dict[str, str]]] = None,
         content_type: Optional[str] = None,
     ):
```

Comment:
```suggestion
    ) -> HTTPResponse:
```

How about adding a type hint as well?